plugin/src/rules/array_callback_return.rs

```diff
@@ -116,7 +116,7 @@ fn full_method_name(array_method_name: &str) -> String {
 pub fn array_callback_return_rule() -> Arc<dyn Rule> {
     rule! {
         name => "array-callback-return",
-        languages => [Javascript],
+        languages => [Javascript, Typescript],
```

Comment:
So now rule tests should iterate through all `SupportedLanguageLanguage`'s of all `languages` declared on the rule (so run each rule test n times against the different grammars, and I believe only explicitly fail if it couldn't parse the query for at least one `SupportedLanguageLanguage` of each of these `languages`)

I should basically just go through and add this to all of the rules that typescript-eslint doesn't override I guess